mibs/algcom/ALGSMXS-v0-MIB

```diff
@@ -0,0 +1,14 @@
+ALGSMXS-v0-MIB DEFINITIONS ::= BEGIN
```

Comment:
shortest mib i ever seen!

Reply:
The attached snmprec is the entire walk of `.` too :D

Reply:
The ALMOST made a table, but didn't quite make it.